gems/zipruby/CVE-2012-1163.yml

```diff
@@ -0,0 +1,15 @@
+---
+gem: zipruby
+cve: 2012-1163
+url: https://packetstormsecurity.com/files/cve/CVE-2012-1163
+title: |
+  libzip (version <= 0.10) has two vulnerabilities that may lead to a heap
+  overflow or an information leak via corrupted zip files
+date: 2012-03-21
+description: |
+  libzip (version <= 0.10) has two vulnerabilities that may lead to a heap
+  overflow or an information leak via corrupted zip files. PHP (versions
+  5.4.0 and <= 5.3.10) and the Ruby binding zipruby (version <= 0.3.6) are
+  also affected as they include copies of affected libzip versions.
+patched_versions:
+  - ">= 0.3.6"
```

Comment:
By that I mean the latest commit in the repository is 2 days before the official release notice of libzip 0.10. Did they actually update it? I don't know much about C...
<img width="266" alt="screen shot 2015-10-22 at 7 41 56 pm" src="https://cloud.githubusercontent.com/assets/5092286/10664316/043216c2-78f5-11e5-8b7d-9d4d48b5d664.png">
<img width="1103" alt="screen shot 2015-10-22 at 7 42 02 pm" src="https://cloud.githubusercontent.com/assets/5092286/10664317/04341c42-78f5-11e5-9999-45d26774bddf.png">